geopandas/explore.py

```diff
@@ -302,7 +307,10 @@ def my_colormap(value):  # scalar value defined in 'column'
             fit = False
 
         # get a subset of kwargs to be passed to folium.Map
-        map_kwds = {i: kwargs[i] for i in kwargs.keys() if i in _MAP_KWARGS}
+        map_kwds = {
```

Comment:
```suggestion
        for i in _MAP_KWARGS:
            if i in map_kwds:
                raise ValueError(
                    f"'{i}' cannot be specified in 'map_kwds'. "
                    f"Use the '{i}={map_kwds[i]}' argument instead."
                )
        map_kwds = {
```

We don't want to mistakenly override the argument. You could in theory specify `df.explore(zoom_control=True, map_kwds={'zoom_control': False})`. But in that case, we have an ambiguity of which has the preference. So I'd check for that and raise an error to avoid it.